<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="allButton">
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="serialNoButton">
        <property name="text">
         <string>Serial No</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="itemTypeButton">
        <property name="text">
         <string>Item Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="locationButton">
        <property name="text">
         <string>location</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="userButton">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="readSingleButton">
        <property name="text">
         <string>Read Single</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="readMultiButton">
        <property name="text">
         <string>Read Multiple</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopReadButton">
        <property name="text">
         <string>Stop Read</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="manageItemsButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>MANAGE ITEMS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rfidToolsButton">
        <property name="text">
         <string>RFID TOOLS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modifyButton">
        <property name="text">
         <string>MODIFY LOCATION</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logButton">
        <property name="text">
         <string>LOG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="findTagButton">
        <property name="text">
         <string>FIND TAG</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>578</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="reloadButton">
        <property name="text">
         <string>Reload</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="messages">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
